src/DynamoCoreWpf/ViewModels/Core/ConnectorPinViewModel.cs

```diff
@@ -92,11 +92,11 @@ public ConnectorPinModel Model
         /// so we only need one property that can be used for both.
```

Comment:
Can you also specify meaning of this name? I guess this is because we are now using the 48px version of icons

Reply:
What name, ConnectorPinModel? Or `OneThirdWidth`?

Reply:
We are using 48x icon, but we are sizing them to 30px typ... So the ConnectorPinModel width and height should still be 30

Reply:
Let me double check to make sure we are setting the height/ width to 30 properly.

Reply:
@QilongTang I am pretty sure I am constraining the height of the icons to 30px, as the parent usercontrol hosting them has  those contraints. But the icon does look bigger, it looks like a proper 48px.

Reply:
Modified the button/ image controls where icon is displayed- now they are consistent.

Reply:
Also updated the summary of the method, per your original comment.